--- google-earth-6.0.1.2032/lib/googleearth~	2010-12-29 01:10:51.000000000 +0200
+++ google-earth-6.0.1.2032/lib/googleearth	2010-12-29 01:12:24.927165032 +0200
@@ -40,4 +40,12 @@
 
 cd $script_path;
 
-LD_LIBRARY_PATH=.:$LD_LIBRARY_PATH ./googleearth-bin "$@"
+# WARNING!
+#
+# This is a workaround for GoogleEarth bug. With some locales GE uses comma as
+# decimal separator while parsing KML files. It renders virtually all data
+# unusable. Ensure the LC_NUMERIC is set to US standard (dot-separated).
+#
+LC_NUMERIC="en_US"
+
+LD_LIBRARY_PATH=.:$LD_LIBRARY_PATH exec ./googleearth-bin "$@"
